packages/fast-components-react-base/src/breadcrumb/breadcrumb.spec.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, mount, shallow } from "enzyme";
+import Breadcrumb, {
+    BreadcrumbClassNameContract,
+    BreadcrumbUnhandledProps,
+} from "./breadcrumb";
+
+/*
+ * Configure Enzyme
+ */
+configure({ adapter: new Adapter() });
+
+const managedClasses: BreadcrumbClassNameContract = {
+    breadcrumb: "breadcrumb-class",
+    breadcrumb_item: "breadcrumb-item-class",
+    breadcrumb_item__current: "breadcrumb-item-current-class",
+    breadcrumb_itemsContainer: "breadcrumb-items-container-class",
+    breadcrumb_separator: "breadcrumb-separator-class",
+};
+
+describe("breadcrumb", (): void => {
+    test("should have a displayName that matches the component name", () => {
+        expect((Breadcrumb as any).name).toBe(Breadcrumb.displayName);
+    });
+
+    test("should not throw if managedClasses are not provided", () => {

```

Comment:
you should add an expect statement with a breadcrumb that implements children and a separator also - with the current implementation it would throw